CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Changes to Calva.
 
 ## [Unreleased]
 
```

Comment:
I think that the script we have for picking out the changes is still picky about no new-line here.

Reply:
Fixed!!